test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -266,4 +285,90 @@ public void restartTest() throws Exception {
         assertThat(corfuRuntime.getLayoutView().getLayout().getEpoch()).isEqualTo(l.getEpoch() + 1);
         assertThat(shutdownCorfuServer(corfuServer)).isTrue();
     }
+
+    /**
+     * Kill a node containing the head of the log unit chain and the primary sequencer.
+     * Ensures writes into after kill node and cluster re-stabilization.
+     *
+     * @throws Exception
+     */
+    @Test
```

Comment:
We should also add a test for the case when a non-primary sequencer log unit is killed.

Reply:
Added.